### DISCUSSION TITLE
table-pin-cols not work

### DISCUSSION THREAD

**Discussion by mftaa** [saadeghi/daisyui#2006] [Q&A] [answered]
hi, i use daisyui in my next js project. when i make a table with table pin cols wish that the column can auto scroll when it exceeds the limit, but it's not working. my table doesn't have auto scroll. am i wrong? what should i do

**Comment by saadeghi** [comment]
Can you please share a code example (Tailwind Play or StackBlitz or a GitHub repo)?

**Reply by mftaa** [reply]
i just copied from daisyui example

**Reply by mftaa** [reply]
I can make it by giving overflow in the class, but for the head of the table it can't be sticky

**Reply by saadeghi** [reply, marked as answer]
Did you set a height? because otherwise, by default, table has no height limit. it just fills all the empty space and when it's filling the space, there is no need for a row to be sticky.  
If you add a height size you can see the sticky row and if you add a width size you can see the sticky column:  
https://play.tailwindcss.com/5bvndLAknD

Or if you want the table to fill the screen height, you can add `max-h-screen` class:  
https://play.tailwindcss.com/RkvogY8frm
